stories/setup/styles.scss

```diff
@@ -48,6 +48,11 @@ body.fullscreen {
   }
 }
 
+.DiffViewStory-panel {
+  border-radius: $border-radius;
+  border: 1px solid $light-gray;
+}
+
 .CodeView-panel {
```

Comment:
This does not look consistent with the rest. I know it has nothing to do with this patch, though.

Reply:
That's true. I filed https://github.com/mozilla/addons-code-manager/issues/692